docs/fluent-registration-api-extensions.md

```diff
@@ -65,26 +65,6 @@ container.Register(
 );
 ```
 
-### What does ActAs do?
```

Comment:
Did you forget to remove `ActAs` from `ComponentRegistration`?

Reply:
Must have accidentally omitted that commit when I squished everything.

Reply:
All fixed